### ISSUE TITLE
Can't upload to remote lotus node

### ISSUE THREAD

**Issue by morrigan** [filecoin-project/starling#61]
I have starling installed locally while my miner runs on remote server.

I have setup API's url and authorization token in the files but upload using `starling store` doesn't seem to work still. Would be nice if starling could work remotely.

The error that I get is `no such file or directory` while file exists locally on my PC.

I'm not sure if this is starling's best use case but would be nice if to have this possible.

**Comment by alexandrumatei36** [comment]
Indeed, as stated in the README file, Starling's current version doesn't support upload from a remote node due to some Lotus limitations
